config/config.go

```diff
@@ -222,11 +224,38 @@ var baseConfig = Config{
 
 var defaultConfig Config
 
+func setupFlags() {
```

Comment:
Lot of these values have been changed with recent commits. @codeasashu can you please resolve accordingly.
Rest LGTM.

Reply:
Sure, will do shortly

Reply:
rebased